src/ophys_etl/transforms/convert_rois.py

```diff
@@ -14,6 +14,7 @@
 from ophys_etl.transforms.roi_transforms import (binarize_roi_mask,
                                                  coo_rois_to_lims_compatible,
                                                  suite2p_rois_to_coo)
+from ophys_etl.filters.filters import filter_longest_edge_length
```

Comment:
Isn't a filter a type of transform? And this particular transform is applied to ROIs...

So, I think the `ophys_etl.filters.filters.filter_longest_edge_length` function might make more sense in 
`ophys_etl.transforms.roi_transforms`?

Reply:
I didn't include it in that file because those add exclusion labels whereas this just removes them from processing all-together. I think the behavior is different enough to justify a new module, @djkapner @kschelonka what do the two of you think?

Reply:
I don't have a super strong opinion. I think if we don't put it in `roi_transforms` we need to change so that it's not `ophys_etl.filters.filters` because that's unnecessarily repetitive

Reply:
I imported the filters.filter_longest_edge_length into the filters module __init__.py file, this required me adding a noqa F401 to the file as it said it wasn't used. This removes the excess filters.filters from the imports in other modules. Let me know if there's a more efficient way to do this.

Reply:
Ok, that gets around the import, but I don't understand why you need another module. Can't everything just live in `ophys_etl.filters.py`?

Reply:
It absolutely can, I had the thought it fit better with the structure of the existing repo since the same can be said about the pipelines module or the qc module, since they're only one file. If it's preferred to move it to just a file I'm fine with that.

Reply:
If you anticipate that we would have an entire module of filters, then that would make sense. But in that case, rename `filters.py` to something that is more descriptive about the type of filters we contain. Otherwise `ophys_etl/filters.py` should be fine.

Reply:
I moved the `filters.py` to the `ophys_etl` module